README.md

```diff
@@ -11,6 +11,9 @@ Google and Facebook Auth
 SSR Ready
 Svelte v3 + v2
 
+## installation
+````npm i @beyonk/svelte-social-auth````
```

Comment:
Svelte applications are bundled, so dependencies should **always** be installed as devDependencies - you should add -D to this command.

Reply:
@antony its a direct copy of the install instructions on npm.  and perhaps also why its failing SSR build on sapper. 

again a good reason to have install instructions.

Reply:
@antony it was the --save-dev switch that was needed to make this work as an SSR component out of the box